frontend/simple/views/SignUp.vue

```diff
@@ -17,17 +17,18 @@
             <h2 class="subtitle">Sign Up</h2>
 
             <p class="control has-icon">
-              <input v-form-ctrl class="input" name="name" pattern="\S*" placeholder="username" required>
+              <input class="input" name="name" v-validate data-rules="required|regex:^\s*\S+\s*$" placeholder="username" required>
```

Comment:
That left usernames like 'me you' or 'me ' possible I did some testing on it before I made this change